src/record/import/utils/iterator.ts

```diff
@@ -0,0 +1,87 @@
+type Iterators<T = unknown> = IterableIterator<T> | Generator<T>;
+type AsyncIterators<T = unknown> = AsyncIterableIterator<T> | AsyncGenerator<T>;
+
+// eslint-disable-next-line func-style
+export async function* iterToAsyncIer<T = unknown>(
```

Comment:
@mshrtsr 

Seem It's a typo issue, isn't it?
`iterToAsyncIer` => `iterToAsyncIter`

Reply:
THX! fixed!